HelpSource/Classes/Float.schelp

```diff
@@ -65,3 +90,69 @@ pi.asStringPrec(6)
 7.4.asStringPrec(5)
 7.4.asStringPrec(50)
 ::
+
+
+log2(inf.asInteger)
+2 ** 31 - inf.asInteger
```

Comment:
what is this doing here..? should it at least be in a code:: block?

Reply:
sorry, leftovers.